<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgStructureClass</class>
 <widget class="QDialog" name="DlgStructureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1334</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Структура станков</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="lwTableName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnInsertRecord">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="resources/rajournal.qrc">
         <normaloff>:/add.png</normaloff>:/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteRecord">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="resources/rajournal.qrc">
         <normaloff>:/trash.png</normaloff>:/trash.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сохранить изменения</string>
       </property>
       <property name="icon">
        <iconset resource="resources/rajournal.qrc">
         <normaloff>:/save.png</normaloff>:/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tvTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="gbxFilter">
     <property name="title">
      <string>Фильтр</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblFilter1">
        <property name="text">
         <string>Модель:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFilter1"/>
      </item>
      <item>
       <widget class="QLabel" name="lblFilter2">
        <property name="text">
         <string>Узел:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFilter2"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lwTableName</tabstop>
  <tabstop>tvTable</tabstop>
  <tabstop>btnInsertRecord</tabstop>
  <tabstop>btnDeleteRecord</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="resources/rajournal.qrc"/>
 </resources>
 <connections/>
</ui>
